tests/server/server-executeCommand-002.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\Server::executeCommand() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$rp = new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY);
+$secondary = $manager->selectServer($rp);
+
+$command = new MongoDB\Driver\Command(array('profile' => 2));
+$cursor = $secondary->executeCommand(DATABASE_NAME, $command);
+$result = current($cursor->toArray());
+
+if (empty($result->ok)) {
+    exit("Could not set profile level\n");
+}
+
+$command = new MongoDB\Driver\Command(array(
+    'aggregate' => COLLECTION_NAME,
+    'pipeline' => array(array('$match' => array('x' => 1))),
+));
+$secondary->executeCommand(DATABASE_NAME, $command, $rp);
+
+$query = new MongoDB\Driver\Query(
+    array(
+        'op' => 'command',
+        'ns' => DATABASE_NAME . '.$cmd',
+    ),
+    array(
+        'sort' => array('ts' => -1),
+        'limit' => 1,
+    )
+);
+$cursor = $secondary->executeQuery(DATABASE_NAME . '.system.profile', $query, $rp);
+$profileEntry = current($cursor->toArray());
+
+var_dump($profileEntry->command);
+
+$command = new MongoDB\Driver\Command(array('profile' => 0));
+$cursor = $secondary->executeCommand(DATABASE_NAME, $command);
+$result = current($cursor->toArray());
+
+if (empty($result->ok)) {
+    exit("Could not set profile level\n");
+}
```

Comment:
There is still a `===DONE===` line below, so I don't see the concern with this inadvertently passing on a segfault.